deploy/packer/base/scripts/packages.sh

```diff
@@ -2,9 +2,9 @@
 
 set -e -v
 
-DOCKER_VERSION=5:18.09.5~3-0~ubuntu-bionic
-KERNEL_VER=4.15.0-47-generic
-V4L2LOOPBACK_VERSION=0.12.0
+DOCKER_VERSION=18.06.3~ce~3-0~ubuntu
+KERNEL_VER=4.4.0-1014-aws
```

Comment:
will this work in GCP and Packet?

Reply:
I guess the question behind that would be: has this built successfully for those clouds as well?

Reply:
packet has a different build process, for it we will stick to Ubuntu 18.04. I didn't test it on GCP, but for EC2 it works.

Reply:
Please test on GCP :)

Reply:
Ok, update the patch to build GCP successfully